pkg/match/path.go

```diff
@@ -15,14 +22,16 @@ import (
 )
 
 const (
-	separatorChars   = `.\-_ `
-	separatorPattern = `(?:_|[^\p{L}\w\d])+`
+	separatorChars       = `.\-_ `
+	separatorPattern     = `(?:_|[^\p{L}\w\d])+`
+	separatorPatternExif = `(?:_|[^\p{L}\w\d\,])+`
```

Comment:
Slightly different pattern because tags are separated by comma so we must not remove them